Compile error while import from "src/orders/types";

### Component

API

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

6.1.4

### What function is the bug in?

_No response_

### Operating System

macOS (Apple Silicon)

### Describe the bug

node_modules/opensea-js/lib/api/types.d.ts:2:30 - error TS2307: Cannot find module 'src/orders/types' or its corresponding type declarations.

`import { ProtocolData } from "src/orders/types";`

If I change `src/orders/types` to `../orders/types`, the error disappeared.

--- a/src/api/types.ts
+++ b/src/api/types.ts
@@ -1,5 +1,5 @@
 import { ConsiderationItem } from "@opensea/seaport-js/lib/types";
-import { ProtocolData } from "src/orders/types";
+import { ProtocolData } from "../orders/types";
 
 export type BuildOfferResponse = {
   partialParameters: PartialParameters;

